src/components/matrixCall/MatrixCallView.styles.scss

```
@@ -1,243 +1,252 @@
+	}
+
+	&__avatar-circle {
+		width: 160px;
+		height: 160px;
+		border-radius: 50%;
+		background: linear-gradient(
+			135deg,
+			#8b4789 0%,
+			#6b4786 100%
+		); // Purple gradient
+		color: #f8d7da; // Light pink text
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)